components/cam/bld/configure

```diff
@@ -128,7 +128,6 @@ OPTIONS
 			  linoz_mam4_resus_mom | linoz_mam4_resus_mom_soag |
                           trop_strat_soa | trop_strat_mam3 | trop_strat_mam7 | none ].  Default: trop_mam3.
      -clubb_sgs         Turns on CLUBB_SGS
-     -shoc_sgs          Turns on SHOC_SGS
```

Comment:
just noting locations where shoc/p3 might be removed.

Reply:
we need to retain this line for SCREAM